src/test/java/org/breedinginsight/brapps/importer/ExperimentFileImportTest.java

```diff
@@ -216,7 +216,7 @@ public void importNewExpNewLocNoObsSuccess() {
 
     @Test
     @SneakyThrows
-    public void importNewExpMultiNewEnvNoObsSuccess() {
+    public void importNewExpMultiNewEnvError() {
```

Comment:
This test asserts that both environments import successfully, so it's not clear to me why the test name needs to change to include "Error".